app/ports/templates/ports/stats_port_installations.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'ports/layout.html' %}
+
+{% load staticfiles %}
+
+{% block head_scripts %}
+    <script src="{% static 'js/ajax-port-installations.js' %}"></script>
+{% endblock %}
+
+{% block title %}Port Installations |{% endblock %}
+
+{% block content %}
+    <br>
+    <ul class="nav nav-tabs" id="tabs" role="tablist">
+        <li class="nav-item">
+            <a href="{% url 'stats_home' %}" class="nav-link">System Stats</a>
+        </li>
+        <li class="nav-item">
+            <a href="#" class="nav-link active">Port Installations</a>
+        </li>
+    </ul>
+    <br>
+    {% csrf_token %}
+    <input type="text" class="form-control" id="search-by" name="search-by" placeholder="Filter by port name">
+    <br>
+    <form class="form-inline">
+    <label class="form-inline" for="days-filter">Showing data for users who submitted stats in last: </label>
+    <select class="form-control form-inline ml-2" id="days-filter" name="days-filter">
+        <option value="7">7 days</option>
+        <option value="30" selected>30 days</option>
+        <option value="90">90 days</option>
+        <option value="180">180 days</option>
+        <option value="365">365 days</option>
```

Comment:
You had some generic code for this elsewhere?

Reply:
Thank you, I have removed the repetition.